src/Console/CommandRunner.php

```diff
@@ -50,6 +58,26 @@ public function __construct(BaseApplication $app, $root = 'cake')
     {
         $this->app = $app;
         $this->root = $root;
+        $this->aliases = [
+            '--version' => 'version'
+        ];
+    }
+
+    /**
+     * Replace the alias map for a runner.
+     *
+     * Aliases allow you to define alternate names for commands
+     * in the collection. This can be useful to add top level switches
+     * like `--version` or `-h`
+     *
+     * @param array $aliases The map of aliases to replace.
+     * @return $this
+     */
+    public function setAliases(array $aliases)
+    {
+        $this->aliases = $aliases;
```

Comment:
Instead of renaming this method to ``replaceAliases()`` we could have ``addAlias()`` /  ``addAliases()`` and ``removeAlias()`` / ``removeAliases()``. This way, it could be clearer that ``setAliases()`` replaces all aliases previously set.